testing/chainsaw/e2e-other/cluster-migrate/05--cluster.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: cluster-migrate
+spec:
+  dataSource:
+    volumes:
+      pgDataVolume:
+        pvcName: data-non-crunchy-cluster-0
+        directory: data
+  postgresVersion: ${KUTTL_PG_VERSION}
```

Comment:
The `generate-kuttl` Make target in the [workflow](https://github.com/CrunchyData/postgres-operator/pull/3844/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R243) is used to fill out these variables with certain variables that might be different between tests.  We would need to update/duplicate that target for chainsaw or figure out a different way to use variables.

@eddycharly Does chainsaw have a concept of variables or variable expansion?

Reply:
`v0.1.3` introduced support to pass arbitrary values to tests and `v0.1.4` will come with a solution to inject values in the resource manifests so basically no need for a generate step anymore :)